plugins/inputs/jboss/jboss.go

```diff
@@ -0,0 +1,1312 @@
+package jboss
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
```

Comment:
Please don't use `log` but the internal `telegraf.Logger` by adding 
```golang
Log telegraf.Logger `toml:"-"`
```
to your plugin struct and the use `h.Log.Debugf()` etc.